app/controllers/subjects_controller.rb

```diff
@@ -88,6 +96,15 @@ def load_link_attempt
     )
   end
 
+  def load_defendant
+    prosecution_case = CourtDataAdaptor::CaseSummaryService.call(@application.case_summary.prosecution_case_reference)
+    @defendant = prosecution_case.defendant_summaries.find do |defendant|
+      defendant.application_summaries.any? do |as|
+        as.id == @application.application_id
+      end
+    end
+  end
+
```

Comment:
I am not a big fan of this `load_...` pattern:
- It obfuscates the vars that are passed into the views
- It adds additional logic in the controllers 
- It's hard to test

An alternative is to just set the var in the action:
```
@defendant = get_defendant_by_prosecution_case(@application.case_summary.prosecution_case_reference)
```
and move `get_defendant_by_prosecution_case` to a "better" place, maybe inside the `prosecution_case` ? 🤔 
`prosecution.get_defendant`

Reply:
Fair point. I've added `.defendant` as a method on ApplicationSummary, and at that point we don't need to set an ivar at all, we can simply call the method when needed in the view.